util/build_docs.py

```diff
@@ -191,9 +192,17 @@ def generate_hardware_blocks():
 
 def generate_testplans():
     for testplan in config["testplan_definitions"]:
+        # Split the filename into filename and tags, if provided.
+        split = str(testplan).split(":")
```

Comment:
Can `testplan` ever be anything other than a string? If not, maybe this should be an assertion, rather than a conversion?